deck.py

```diff
@@ -3,9 +3,9 @@
 
 
 class Deck:
-    suits = ["Clubs", "Hearts", "Spades", "Diamonds"]
+    suits = ["\u2663", "\u2665", "\u2660", "\u2666"]
 
-    values = ["2", "3", "4", "5", "6", "7", "8", "9", "10", "Jack", "Queen", "King", "Ace"]
+    values = range(1, 14)
```

Comment:
You want to be careful with doing something like this. `range` returns an iterator. Which traditionally is a one time use list. However, `range` is a special kind that lets you iterate over it as many times as you want. Long story short, this is fine, but be cognizant about the ramifications of iterators vs lists.

Reply:
So, `range(1, 4)` is not the same as `[1, 2, 3]`. I can understand that, but I don't know how else I would accomplish this. It seems silly to write a list containing all the numbers 1-13.

Reply:
That makes sense. Thank you